public/template_app_role_upload.csv

```diff
@@ -1,6 +1,6 @@
-﻿roles;description
-Administrator;Administrator are able to configure the application as per the costumer need
-Commercial Specialist;Commercial Specialist can approve commercial request and send them for the manager review
-Commercial Manager;Commercial Specialist can do the final approval of commercial request and trigger the payout.
-Recycler;Recycler can buy recycled material
-Manager;This is only a test text
\ No newline at end of file
+﻿roles;description_en;description_de
+"Administrator;Administrator are able to configure the application as per the costumer need;Administratoren sind in der Lage, die Anwendung nach den Bedürfnissen der Kunden zu konfigurieren."
+Commercial Specialist;Commercial Specialist can approve commercial request and send them for the manager review;Der Handelsspezialist kann Handelsanfragen genehmigen und sie zur Überprüfung durch den Manager senden.
+Commercial Manager;Commercial Specialist can do the final approval of commercial request and trigger the payout;Der Handelsspezialist kann die endgültige Genehmigung des Handelsantrags vornehmen und die Auszahlung veranlassen.
+Recycler;Recycler can buy recycled material;Recycler können recyceltes Material kaufen
+Manager;This is only a test text in EN;Das is in Deutsch
```

Comment:
the translations should not appear in this template. Instead we should have them together with the other translations in src/assets/locales.